### ISSUE TITLE
Display results in a separate tab

### ISSUE THREAD

**Issue by dbamaster** [Microsoft/azuredatastudio#4784]
**Is your feature request related to a problem? Please describe.**
I like to have the query results displayed into a separated tab because it maximizes the space I can use to work when writing a query

**Describe the solution or feature you'd like**
Been able to display results in a separate tab

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by sommarskog** [comment]
I can only say that this is long overdue. This is how the 6.5 tools worked. This was in Query Analyzer starting with QA 2000, and it always been in SSMS. There is all reason to put this into ADS. This is absolutely a productivity booster. The split window with Query on top and Results at bottom introduced by the original Query Analyzer in SQL 7 is an awfully bad compromise. I see too little of the query, too little of the results.

**Comment by erinstellato-ms** [comment]
@sommarskog and @dbamaster There is a query option to do this is in SSMS _(Tools | Options, Query Results | SQL Server | Results to Grid | Display results in a separate tab.

Do you use that in SSMS, and if you had that as an option in ADS, would that be sufficient?  Or are you asking for the default behavior to be a separate tab?

**Comment by sommarskog** [comment]
Yes, I use this in SSMS, always. I think this should be an option. Given that many people are not acquainted with this behaviour, it would be brave to make it the default. Very brave.

**Comment by dbamaster** [comment]
Hey Erin! That's correct, my suggestion was based upon this existing option in SSMS. Sending those results to a separate tab by default would be ideal (to me).

**Comment by erinstellato-ms** [comment]
@sommarskog @dbamaster Thank you both for your feedback and the clarification!  I understand that it may be ideal as a default for some, but I agree with Erland that making this behavior a new default would be very brave ;) I will see what we can do to get this option added.
